fsharp-backend/tests/FuzzTests/FQFnName.FuzzTests.fs

```diff
@@ -32,7 +32,7 @@ let modName : Gen<string> = nameGenerator [ 'A' .. 'Z' ] G.alphaNumericString
 let fnName : Gen<string> = nameGenerator [ 'a' .. 'z' ] G.alphaNumericString
 
 type Generator =
-  static member SafeString() : Arbitrary<string> =
+  static member String() : Arbitrary<string> =
```

Comment:
Could these be included via inheritence as well, like the NodaTime ones (or is there only single inheritence, I forget)?

Reply:
Yes - will do